ephios/plugins/complexsignup/signals.py

```diff
@@ -12,3 +14,20 @@ def register_complex_shift_structures(sender, **kwargs):
     return [
         ComplexShiftStructure,
     ]
+
+
+@receiver(nav_link, dispatch_uid="ephios.plugins.complexsignup.signals.nav_link")
+def add_nav_link(sender, request, **kwargs):
+    return (
+        [
+            {
+                "label": _("Signup blocks"),
+                "url": reverse_lazy("complexsignup:blocks_editor"),
+                "active": request.resolver_match
+                and request.resolver_match.app_name == "complexsignup",
+                "group": _("Plugins"),
```

Comment:
I do like it, but I don't think "Plugins" is a helpful name. Also, while I do think Qualifications surpass a threshold that legitimizes its placement in the settings, but it's all very close.